core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5625,7 +5625,27 @@ components:
             Sam.
         managedAppCoordinates:
           $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        members:
+          description: The list of members to add to the billing project
+          type: array
+          items:
+            $ref: '#/components/schemas/ProjectAccessUpdate'
+        inviteUsersNotFound:
+          description: If true, invite the users to Terra if they are not registered in the system. They will be given access to the project
+          type: boolean
       description: ""
+    ProjectAccessUpdate:
+        required:
+          - email
+          - role
```

Comment:
Can you include what the supported roles are? I would hope we already have an enum for this someplace in the documentation.

Reply:
Added an enum in 17eac898607fd7ff7679bf2fdeb92e01d0174ce8 and updated a few other existing places to use it